src/app/components/Drbb/DrbbItem.jsx

```diff
@@ -70,6 +81,25 @@ const DrbbItem = (props) => {
     );
   };
 
+  const downloadLinkElement = () => {
+    if (!downloadLink) return null;
+    if (!downloadLink.download) return null;
+
+    return (
+      <Link
```

Comment:
This is a subtle thing, but the download link is actually a button that looks like a link, so it should be a semantic `<button>` rather than `<a>`